src/resources/assets/js/components/enso/bulma/BooleanFilter.vue

```diff
@@ -2,6 +2,7 @@
 
     <vue-filter
             :options="options"
+            :icons="true"
```

Comment:
`icon` does the same thing as `:icons=true`, being a boolean flag that defaults to false.